tests/test_fyle/test_tasks.py

```
@@ -490,3 +502,966 @@ def test_skip_expenses_and_post_accounting_export_summary(mocker, db):
+
+    recreate_expense_groups(workspace_id=workspace_id, expense_ids=expense_ids)
+
+    remaining_expenses = Expense.objects.filter(id__in=expense_ids, workspace_id=workspace_id)
+    assert remaining_expenses.count() == 0
+
+
+
+
+
+
+def test_delete_expenses_in_db(mocker, db):
```

Ruff (0.13.1)

Unused function argument: `db`

(ARG001)